private_sharing/models.py

```diff
@@ -405,6 +405,7 @@ class Meta:  # noqa: D101
                      { 'project_member_id': '12345678', 'erasure_requested': True}""",
         verbose_name="Deauthorization Webhook URL",
     )
+    log_api_access = models.BooleanField(default=False)
```

Comment:
per discussion, let's go with the `settings` approach with an `.env` variable instead